addon/mixins/unauthenticated-route-mixin.js

```diff
@@ -62,7 +62,6 @@ export default Mixin.create({
    `beforeModel` method is actually executed.
 
     @method beforeModel
-    @param {Transition} transition The transition that lead to this route
```

Comment:
I don't think we should be removing this - this method actually accepts a `transition` argument although that might no longer be used.